src/ext/legit/legit.lisp

```diff
@@ -291,14 +295,21 @@ Next:
   to the user on success as a tooltip message,
   or show the external command's error output on a popup window.
 
+  The function FN returns up to three values:
+
+   - standard output (string)
+   - error output (string)
+   - exit code (integer)
+
   Use with-current-project in the caller too.
   Typicaly used to run an external process in the context of a diff buffer command."
   (multiple-value-bind (output error-output exit-code)
       (funcall fn)
-    (declare (ignorable output))
     (cond
       ((zerop exit-code)
-       (message (str:join #\newline (remove-if #'null (list message output)))))
+       (let ((msg (str:join #\newline (remove-if #'null (list message output)))))
+         (when (str:non-blank-string-p msg)
+           (message msg))))
```

Comment:
It probably malfunctions if a `~` is included.

Reply:
mmh right, but the messages come from porcelain, so it should be fine?

Reply:
Hmm, I think it's better to write in a safe way to avoid unnecessary worry.

Reply:
That said, it's not enough of a problem to hinder development.
I am very grateful to you for doing such a great job.